plugin-3.0/pom.xml

```diff
@@ -0,0 +1,96 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+  <groupId>com.helger.maven</groupId>
+	<artifactId>maven-jaxb30-plugin</artifactId>
+	<packaging>maven-plugin</packaging>
+	<name>Maven JAXB 3.0.x Plugin</name>
+	<parent>
+		<groupId>com.helger.maven</groupId>
+		<artifactId>maven-jaxb2-plugin-project</artifactId>
+		<version>0.14.4-SNAPSHOT</version>
```

Comment:
I even suggest that the addition of JAXB 3.0 would suffice the requirements for a 0.15.0 version :)

Reply:
If you agree, I would do this afterwards with the "mvn versions:set" plugin

Reply:
Yes that sounds sensible to me.